tools/taskcluster/tasks.py

```diff
@@ -91,39 +91,45 @@ def craft_release_build_task(self, tag):
             chain_of_trust=True,  # Needed for sign and push task verification
         )
 
-    def craft_email_task(self, sign_task_id, tag):
+    def craft_email_task(self, sign_task_id, push_task_id, tag):
+        # The "\\n" are hard-coded on purpose, since we don't want a newline in the string, but
+        # we do want the JSON to have the escaped newline
+        content = 'Automation for this release is ready. Please: \\n' \
+                  f'* Download the APK and attach it to the [Github release](https://github.com/mozillamobile/firefox-tv/releases/tag/{tag})\\n' \
+                  '* [Deploy the new release on Amazon](https://developer.amazon.com/apps-and-games/console/app/amzn1.devportal.mobileapp.7f334089688646ef8953d041021029c9/release/amzn1.devportal.apprelease.4ca3990c43f34101bf5729543343747a/general/detail)'
         script = f'''
-        apt update
-        && apt install -y python3-pip
-        && pip3 install taskcluster
-        && taskcluster api notify email << EOM
+        curl -X POST "$TASKCLUSTER_PROXY_URL/notify/v1/email" -H "Content-Type: application/json" -d "$(cat <<EOM
         {{
             "address": "{NOTIFY_EMAIL_ADDRESS}",
-            "content": "Download the APK and attach it to the [Github release](https://github.com/mitchhentges/firefox-tv/releases/tag/{tag})",
+            "content": "{content}",
             "link": {{
-                "href": "https://mozilla.com",
+                "href": "https://queue.taskcluster.net/v1/task/{sign_task_id}/artifacts/public/build/target.apk",
                 "text": "{tag} APK"
             }},
-            "subject": "{tag} Github release artifact is signed"
+            "subject": "Release {tag} is ready for deployment"
         }}
         EOM
+        )"
         '''
 
         return self._craft_shell_task(
-            'Email that Github APK is signed',
+            'Email that automation is complete',
             script,
             [f'notify:email:{NOTIFY_EMAIL_ADDRESS}'],
             {},
-            dependencies=[sign_task_id],
+            dependencies=[sign_task_id, push_task_id],
         )
 
     def _craft_shell_task(self, name, script, scopes, artifacts, *, dependencies=(), chain_of_trust=False):
-        trimmed_script = ' '.join([line.strip() for line in script.split('\n') if line.strip()])
+        trimmed_script = '\n'.join([line.strip() for line in script.split('\n') if line.strip()])
```

Comment:
It looks like we remove all the \n from the script string and then rejoin it with \n: should we just use the script string directly?

Reply:
We could use the script string directly, but by `strip()`-ing the extra spacing from the python heredoc values, we make the build tasks simpler (the "command" payload will have less irrelevant whitespace)

Reply:
Maybe comment to clarify that removing whitespace is the intent? I'm surprised there isn't a built-in python function to do this already. :P

Reply:
Good call!